### ISSUE TITLE
ScrollControls scroll Html element even outside of <Scroll> in Typescript

### ISSUE THREAD

**Issue by dpnick** [pmndrs/drei#1048]
Hi everyone!

First of all, thanks for this amazing library, I have some much fun working with it.

- `three` version:
- `@react-three/fiber`: ^8.6.0
- `@react-three/drei`: ^9.23.4
- `node` version: v18.9.0
- `npm` version: 8.19.1

### Problem description:

I recently found a new (I guess) issue with the `ScrollControls`.  As explained in the title, if I put a `Html` element inside the `ScrollControls`, this `Html` element will move according to the scroll position (which is not expected because it is not inside a `Scroll` element).

For some reasons, I was able to reproduce this bug only in Typescript (it works as expected in pure JS).

### Relevant code:

Here is a minimal code to repro:

```js
import { Html, ScrollControls } from '@react-three/drei';
import { Canvas } from '@react-three/fiber';
import React from 'react';
import './App.css';

function App() {
  return (
      <Canvas style={{ position: 'fixed', top: 0, left: 0 }}>
        <ScrollControls pages={6}>
          <Html style={{ background: 'green' }}>
            <p style={{ color: 'red' }}>This is a test</p>
          </Html>
        </ScrollControls>
      </Canvas>
  );
}

export default App;
```

### Suggested solution:

I'm not sure if it is the best solution, but I at least found a temporary one:
- add a `wrapperClass` to your `Html` element with `position: fixed !important`

I also found a potential interesting hint, it seems that for the same code (example above) the component tree is not the same for Typescript and pure JS.

**Comment by yondr-luka** [comment]
https://stackoverflow.com/questions/73661855/react-fiber-drei-scrollcontrols-preventhtml-from-scrolling

**Comment by TomasGonzalez** [comment]
Hi I'm also facing this issue, It used to work just fine in version `^8.10.7` before I updated.

**Comment by michelecocuccio** [comment]
Hello. I have this issue as well. Will it be fixed soon? Thanks

**Comment by TomasGonzalez** [comment]
> Hello. I have this issue as well. Will it be fixed soon? Thanks

I just submitted a PR to fix it [c967c5b](https://github.com/TomasGonzalez/drei/commit/c967c5bdeda0fcc4d77585801b618d28628c23cd), you can go check it out, it was just one line. Can use NPM patch until it is released and continue working.

**Comment by FlySandwich4** [comment]
This bug might appear again. I tried the same code mentioned in the sandbox and the Html text will be scrolled to the top (not fixed with the object) Using the code from: https://codesandbox.io/p/sandbox/bug-drei-scrollcontrols-txccfw?file=%2Fsrc%2Findex.js%3A1%2C1-32%2C1

Really appreciate if this can be fixed.



 "@react-three/drei": "^10.0.6",
  "@react-three/fiber": "^9.1.2",
  "@react-three/xr": "^6.6.13",
  "@splinetool/r3f-spline": "^1.0.2",
  "@types/three": "^0.175.0",
  "framer-motion": "^12.7.3",
  "headlessui": "^0.0.0",
  "heroicons": "^2.2.0",
  "motion": "^12.7.3",
  "next": "15.3.0",
  "react": "^19.0.0",
  "react-dom": "^19.0.0",
  "three": "^0.175.0"
